test/integration/features/cmd-config.feature

```diff
@@ -210,17 +210,8 @@ user defined options which changes default behaviour of Minishift.
      Then stdout should match "Disk size\s*:\s*25 GB"
      Then stdout should match "vCPUs\s*:\s*3"
       And Minishift should have state "Running"
-
-  Scenario: Checking that disk-size value was applied
-     When executing "minishift ssh -- sudo fdisk -l | grep Disk" succeeds
-     Then stdout should match "Disk \/dev\/.da: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
-
-  Scenario: Checking that cpus value was applied
-     When executing "minishift ssh -- cat /proc/cpuinfo" succeeds
-     Then stdout should match "processor\s*: 0"
-      And stdout should match "processor\s*: 1"
-      And stdout should match "processor\s*: 2"
-      And stdout should not match "processor\s*: [3-9]"
+      And the vm should run with "3" cpus
```

Comment:
s/vm/Minishift vm/

Reply:
Also it should be 4 vcpus, isn't it

Reply:
I will do the change to make step specific, However the ```vm``` notation in caps would be more accurate i.e ```VM``` as we have in basic feature file
```
And the Minishift VM should run within "25" to "27" GB of disk size
```

Reply:
One more review comment i saw around the notation ```cpus```, I think some how it got deleted. Anyway i like the comment of whether we should use ```vCPUs``` instead of ```cpus```. IMO ```vCPUs``` will be more accurate as the cups are assigned to the Minishift VM.

Reply:
Done